pallets/crowdloans/src/lib.rs

```diff
@@ -623,7 +668,11 @@ pub mod pallet {
 
             Self::try_mutate_vault(crowdloan, VaultPhase::Succeeded, |vault| {
                 Self::do_withdraw(crowdloan, vault.contributed, VaultPhase::Expired)?;
-                Self::deposit_event(Event::<T>::VaultSlotExpiring(crowdloan));
+                Self::deposit_event(Event::<T>::VaultDoWithdrawing(
```

Comment:
shall this event be deposited directly from `do_withdraw`

Reply:
yeah, it could be archived, I' fix it.